sdks/wasm/sample.cs

```diff
@@ -38,22 +38,13 @@ static void Main(string[] args)
             global = new DOMObject(string.Empty);
             navigator = new DOMObject("navigator");
 
-            var httpMessageHandler = typeof(HttpClient).GetField("GetHttpMessageHandler",
-                                            BindingFlags.Static |
-                                            BindingFlags.NonPublic);
-
             var window = (JSObject)WebAssembly.Runtime.GetGlobalObject("window");
             using (var location = (JSObject)window.GetObjectProperty("location"))
             {
                 BaseApiUrl = (string)location.GetObjectProperty("origin");
             }
 
-            window = null;
-
-            // Setup the HttpClientHandler
-            httpMessageHandler.SetValue(null, (Func<HttpMessageHandler>)(() => {
-                return new WasmHttpMessageHandler();
-            }));
+            window.Dispose();
```

Comment:
Should use using as well

Reply:
That should have been removed altogether but `using` is better.  Will change it.